feature_mining/demo_files/sample_dataset_1_text.csv

```diff
@@ -0,0 +1,38 @@
+doc_id,sentence_id,text,flag_title,feature_removed
+1,1,ipod,1,
+1,2,I like the compact ipod and the features it offered,0,
+1,3,It is handy to carry around because of the  and easy to store,0,3
+1,4,The light weight also makes it easy to move with,0,
+1,5,It works well and I have had no problems with it,0,
+2,6,iPod is Awesome,1,
+2,7,This is my second iPod,0,
+2,8,"My first was a ""mini"" which the nano makes look like a ""jumbo""",0,
```

Comment:
The double quotes are needed there because they are verbatim from the review, so in order for them to show up correctly in the csv as a text string they need to be escaped. They end up being stripped anyway as part of the parsing, but this is how the review was originally written.

Reply:
Ok, clear.